src/controller/controldevice/controller/Controller.cpp

```diff
@@ -73,6 +75,17 @@ bool Controller::HasConfig() {
     return CVarGetInteger(hasConfigCvarKey.c_str(), false);
 }
 
+bool Controller::IsConnected() {
+    const auto deck = Context::GetInstance()->GetControlDeck();
+    for (uint8_t lusIndex = Blue; lusIndex < Max; lusIndex++) {
+        if (this->HasMappingsForLUSDeviceIndex(static_cast<LUSDeviceIndex>(lusIndex))) {
+            return true;
+        }
+    }
+
+    return false;
+}
```

Comment:
this will return true if any mapping exists for for a controller, even if the mapping is to a physical device that is disconnected